### PR TITLE
integration tests enhancing debug capabilities

### PR DESCRIPTION
**Pull request by HarrietwilliamsTAB**
reduced the number of logs outputted when integration tests are run to streamline useful info.

### CODE REVIEW

File: integration-tests/initiateDataRequest.spec.ts [hunk 1]
```diff
@@ -116,7 +117,6 @@ describe('Submit a PII request with approved ticket data', () => {
       assertEventPresent(initiateDataRequestEvents, DATA_SENT_TO_QUEUE_MESSAGE)
 
       const messageId = getQueueMessageId(initiateDataRequestEvents)
-      console.log('messageId', messageId)
```
Comment (rikumolugds) [inline integration-tests/initiateDataRequest.spec.ts:119]: I think the message id is useful for debugging in the logs as it's used to link more than one log stream.

File: integration-tests/initiateDataRequest.spec.ts [hunk 2]
```diff
@@ -241,7 +254,6 @@ describe('Submit a PII request with approved ticket data', () => {
...
```
Comment (rikumolugds) [inline integration-tests/initiateDataRequest.spec.ts:244]: Please leave the message id as per comment above.

File: integration-tests/initiateDataRequest.spec.ts [hunk 3]
```diff
@@ -300,8 +318,6 @@ describe('Submit a PII request with approved ticket data', () => {
 
       const messageId = getQueueMessageId(initiateDataRequestEvents)
 
-      console.log('messageId', messageId)
```
Comment (rikumolugds) [inline integration-tests/initiateDataRequest.spec.ts:303]: Please see similar comment above.

File: integration-tests/initiateDataRequest.spec.ts [hunk 4]
```diff
@@ -342,7 +363,6 @@ describe('Submit a PII request with approved ticket data', () => {
       assertEventPresent(initiateDataRequestEvents, DATA_SENT_TO_QUEUE_MESSAGE)
 
       const messageId = getQueueMessageId(initiateDataRequestEvents)
-      console.log('messageId', messageId)
```
Comment (rikumolugds) [inline integration-tests/initiateDataRequest.spec.ts:345]: Please see above.

File: integration-tests/utils/aws/cloudWatchGetLogs.ts [hunk 5]
```diff
@@ -103,10 +101,6 @@ const waitForEventWithPatterns = async (
     if (logEvents.length > 1) {
       throw Error('More than 1 event matched, check filter patterns')
     }
-
-    const message = logEvents[0]?.message as string
-    console.log(`Found event: ${message}`)
```
Comment (rikumolugds) [inline integration-tests/utils/aws/cloudWatchGetLogs.ts:108]: We'd be interested in the actual event matched for debugging purposes.

**Review by rikumolugds — COMMENTED** [review]
Comments mainly around logging message id for debugging.

File: integration-tests/utils/aws/cloudWatchGetLogs.ts (hunk 5 above)
Comment (HarrietwilliamsTAB) [reply integration-tests/utils/aws/cloudWatchGetLogs.ts:108]: have added those back

Comment (andylangridge) [reply integration-tests/utils/aws/cloudWatchGetLogs.ts:108]: I would argue the opposite. We don't need to log this event as it's massive and is one of the main causes of the noise. Might be more useful to log something generic out e.g. "Event found..." if you want to see where you are in the flow. Or somehow log out the name of the event you are searching for?

**Review by andylangridge — APPROVED** [review]

**Pull request merged**